index.md

```diff
@@ -1,5 +1,51 @@
 ---
+title: What is PlaceOS?
 slug: /
+description: Brief intro to the product
 ---
+<!-- This is a placeholder page in markdown to be replaced post-mvp with a more pleasing React page -->
 
-This is a placeholder for the main landing page.
+PlaceOS is a software platform which makes interaction with physical spaces simple.
+It provides tools to design, build and iterate integrated digital experiences for physical environments.
+
+To do this it merges elements on an API gateway, ESB, and traditional BMS, without the legacy baggage.
+This allows disparate systems, devices and services to combine as a seamless UX for your _place_, whatever that is.
```

Comment:
Might be worth aligning what is said here with https://place.technology/solutions/platform, along with a link back to that page, or the root of the site for more intro info.